package.json

```diff
@@ -33,11 +32,13 @@
     "clean-modules": "rimraf \"node_modules/!(rimraf|.bin)\""
   },
   "devDependencies": {
+    "await-semaphore": "^0.1.3",
```

Comment:
What is this used for? I dont see any usages of this module.

Reply:
It's a dev dependency by device-lister as I recall.